README.md

```diff
@@ -26,50 +26,62 @@ This repository contains:
     - Install/update ARM GCC toolchain:
-
-	      sudo apt autoremove gcc-arm-none-eabi
-          wget https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
-          sudo tar -xvf gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2 -C /opt/
-          echo "PATH=\$PATH:/opt/gcc-arm-none-eabi-9-2019-q4-major/bin" >> ~/.bashrc
-          export PATH=$PATH:/opt/gcc-arm-none-eabi-9-2019-q4-major/bin 
-       **Note:** If you are not using Ubuntu 18.04 and/or bash you may need to modify this script's paths/files. 
-	
-	For Windows
+        ```
+        sudo apt autoremove gcc-arm-none-eabi
+        wget https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2
+        sudo tar -xvf gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2 -C /opt/
+        echo "PATH=\$PATH:/opt/gcc-arm-none-eabi-9-2020-q2-update/bin" >> ~/.bashrc
+        export PATH=$PATH:/opt/gcc-arm-none-eabi-9-2020-q2-update/bin 
+        ```
+        **Note:** If you are not using Ubuntu 18.04 and/or bash you may need to modify this script's paths/files.
+    - Install CMake:
+        * Follow [kitware instructions](https://apt.kitware.com/) to add Latest CMake apt repository
+        * `sudo apt install cmake`
+    
+    For Windows
     - Install [Windows Subsystem for Linux (WSL)](https://linuxconfig.org/how-to-install-ubuntu-18-04-on-windows-10) with Ubuntu 18.04
     - Follow Ubuntu setup instructions (optionally instead of `screen` you can use [PuTTy](https://www.chiark.greenend.org.uk/~sgtatham/putty/), a GUI Windows app)
-	
-	For Mac
+    
+    For Mac (Not Recommended)
     - Open Command Line
-    - Install Homebrew if not installed 
-    	`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+    - Install Homebrew if not installed
+        ```
+        /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        ```
     - Download auto-run script, which will auto install with latest version:
-    	`brew tap ARMmbed/homebrew-formulae`
+        ```
+        brew tap ARMmbed/homebrew-formulae
+        ```
     - Install ARM GCC toolchain via HomeBrew:
-    	`brew install arm-none-eabi-gcc`
+        ```
+        brew install arm-none-eabi-gcc
+        ```
     - Install [ZOC](https://www.emtec.com/zoc/index.html) for serial interfacing
 
 2. Verify the the toolchains were installed correctly
 
     Open a new Command Prompt / Terminal window and run the following commands:
-
-    `make --version                    # Should be v3.8.x or newer`  
-    `arm-none-eabi-gcc --version       # Should be v9.2.x or newer`
+    ```shell script
+    make --version                    # Should be v3.8.x or newer
+    arm-none-eabi-gcc --version       # Should be v9.3.x or newer
```

Comment:
I've followed the instructions for upgrading arm-none-eabi-gcc, but versoin is still 9.2.1. are u sure the steps are right? also, are arm-none-eabi-gcc and gcc-arm-none-eabi(line 37) the same?

Reply:
gcc-arm-none-eabi is just the name of the linux pkg iirc. we remove it cuz we want to use newer versions than what is included with ubuntu 18 (ubuntu locks packages at release date so the version that comes with apt is 2 years old)

Reply:
If you follow the instructions, than it should work fine

Reply:
only thing i can think of is if the older one is picked up on the path and takes priority.... which shouldn't be the case cuz later things in your path take precedence

Reply:
Waht does `echo $PATH` give u

Reply:
/opt/ros/melodic/bin:/home/younes/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/opt/gcc-arm-none-eabi-9-2019-q4-major/bin:/opt/gcc-arm-none-eabi-9-2020-q2-update/bin:/opt/gcc-arm-none-eabi-9-2020-q2-update/bin

Reply:
i tried the steps twice hence it being listed twice. ive removed the line in .bashrc for the old version. should i try to just delete the redudant and old copies?

Reply:
i had to delete the old version but it works now